src/mame/ibm/thinkpad8xx.cpp

```diff
@@ -0,0 +1,92 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***********************************************************************************************************
+Skeleton driver for IBM Thinkpad Power Series.
+The IBM ThinkPad Power Series (800/820/821/822/823/850/851/860) is a laptop series from the ThinkPad line
+that was manufactured by IBM. It is based on the PowerPC architecture.
+All of the PowerPC ThinkPads could run Windows NT 3.51 and 4.0, AIX 4.1.x, and Solaris Desktop 2.5.1 PowerPC
+Edition. It is also possible to run certain PowerPC versions of Linux on the 800 Series.
+
+Hardware for the 850 model:
+-SCSI hard disk.
+-SCSI CD-ROM drive.
```

Comment:
According to https://www.os2museum.com/wp/ibm-thinkpad-power-series-850/
> Storage was unusual to say the least. The ThinkPad 850 used the same **NCR 53C810** SCSI host adapter as the desktop Power Series 440 machine (the 830/850 desktops on the other hand used IDE storage). The NCR 810 was one of the earliest **PCI** SCSI controllers available, which was no doubt one of the reasons why IBM chose it for the PowerPC prototypes in 1993-1994.

Reply:
Yes, the chip was on the back side of the PCB. It's now on the hardware notes.